src/zenml/integrations/mlx/__init__.py

```diff
@@ -21,19 +22,37 @@
 
 # MLX can run on Linux, but only with CPU or Cuda devices,
 # see https://ml-explore.github.io/mlx/build/html/install.html#cuda ff.
+# On macOS, MLX only supports Apple Silicon (ARM64), not Intel (x86_64).
 SUPPORTED_PLATFORMS = ("darwin", "linux")
```

Comment:
I think this can be removed now? It's probably still used in the test, which should now also skip the test depending on the `_is_supported_platform` check instead I assume?